examples/ex1.cpp

```diff
@@ -165,7 +165,8 @@ int main(int argc, char *argv[])
    BilinearForm *a = new BilinearForm(fespace, assembly, elem_batch);
 
    // These will be unified in methods of DiffusionIntegrator
-   if (pa) { a->AddDomainIntegrator(new PADiffusionIntegrator(one)); }
+   PADiffusionIntegrator *paDiffInteg;
+   if (pa) { a->AddDomainIntegrator(paDiffInteg = new PADiffusionIntegrator(one)); }
```

Comment:
Given that this is user facing it might make sense to pull the assignment to it's own line.  Not everyone is familiar with the assignment operator return value.

Reply:
Right, @jdahm sugested me to put that in the `PABilinearFormExtension` instead.

Reply:
👍